DeutschJozsaAlgorithm/DeutschJozsaAlgorithm.ipynb

```diff
@@ -1,8 +1,10 @@
 {
  "cells": [
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
```

Comment:
Looks like all Markdown cells in the notebook somehow got converted into code cells (which, I expect, is causing the errors in CI build, since it's trying to run all code cells). Could you please revert this part of the change?